plugin/googlemeet/resources/css/style.css

```diff
@@ -0,0 +1,148 @@
+.plugin_logo{
+    text-align: center;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+}
+.plugin_logo .img-responsive{
+    margin: auto;
+}
+.tools{
+    margin-bottom: 2rem;
+    margin-top: 1rem;
+}
+.card{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    word-wrap: break-word;
+    border: 1px solid rgba(0,0,0,.05);
+    border-radius: .375rem;
+    background-color: #fff;
+    background-clip: border-box;
+}
+.card-meet{
+    margin-bottom: 30px;
+    border: 0;
+    box-shadow: 0 0 2rem 0 rgba(136,152,170,.15);
+}
+.card-meet:hover{
+    border-left: .25rem solid #cdcdcd !important;
+}
+.card-meet .card-body {
+    padding: 1rem 1.5rem;
+}
+.row-meet {
+    display: block;
+}
+
+.description{
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 2rem;
+    font-size: 12px;
+    color: #666666;
+    display: grid;
+}
+.description .title{
+    font-size: 1.45rem;
+    font-weight: 600!important;
+    color: #32325d;
+    overflow:hidden;
+    white-space:nowrap;
+    text-overflow: ellipsis;
+    width: 100%;
+    display: inline-block;
+}
+.description .text{
+    display: grid;
+    padding-bottom: 1rem;
+}
+.rounded-circle {
+    border-radius: 50%!important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5686281)